packages/bpk-component-navigation-bar/readme.md

```diff
@@ -0,0 +1,56 @@
+# bpk-component-navigation-bar
+
+> Backpack navigation bar component.
+
+## Installation
+
+```sh
+npm install bpk-component-navigation-bar --save-dev
+```
+
+## Usage
+
+```js
+import React from 'react';
+import { withButtonAlignment, withRtlSupport } from 'bpk-component-icon';
+import ArrowIcon from 'bpk-component-icon/sm/long-arrow-left';
+import { BpkButtonLink } from 'bpk-component-link';
+
+import BpkNavigationBar, { BpkNavigationIconButton } from 'bpk-component-navigation-bar';
+
+const AlignedArrowIcon = withButtonAlignment(withRtlSupport(ArrowIcon));
+
+export default () => (
+  <BpkNavigationBar
+    title="Backpack"
+    leadingButton={
+      <BpkNavigationIconButton onClick={() => {}}>
+        <AlignedArrowIcon />
```

Comment:
I think alignment should be handled inside the `BpkNavigationIconButton` component, so should any visually hidden text. Perhaps a `label` prop?

Reply:
Yeah something like that :) or even `icon={Icon}` - that way you wouldn't have to clone the element 😇 .

Consumers already have flexibility by being able to pass anything they want into `leadingButton` i.e.:
```
leadingButton={
  <MyCustomButton />
}
```